### PR TITLE
Set margin-top to remove button in organization members

### LINKED ISSUE #23759: Remove user button in org team page is out of alignment
### Description

image

### Gitea Version

latest

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

build

### Database

None

### PR DESCRIPTION
**Pull request by BElluu**
Fix #23759
image

### CODE REVIEW

**Comment by wxiaoguang** [comment]
Gitea's codebase avoids using inline `style` as much as possible. And I guess it's better to use `flex` instead of `margin` for general alignments.

**Comment by silverwind** [comment]
Adding `git-mt-2` will add 4px `margin-top`. If that is sufficient, use that. Otherwise use real CSS instead of inline style.

**Comment by silverwind** [comment]
And yes, a better fix is to vertically center via flexbox, e.g. add `gt-df dt-ac` to the parent element.

**Comment by BElluu** [comment]
@silverwind @wxiaoguang Thank you for your advice. I used gt-mt-3 instead of margin-top and looks pretty nice :)

**Comment by silverwind** [comment]
It's still off-center, needs to be refactored to flexbox which will require HTML changes as well because the elements are not in the right order in HTML.

image

I will check it out later.

File: templates/org/member/members.tmpl [hunk 1]
```diff
@@ -55,7 +55,7 @@
 							{{end}}
 						</div>
 					{{end}}
-					<div class="ui three wide column">
+					<div class="ui three wide column gt-mt-3">
```
Comment (silverwind) [inline templates/org/member/members.tmpl:58]:
```suggestion
					<div class="ui three wide column gt-df gt-ac gt-je">
```

**Comment by silverwind** [comment]
I ended up reworking this more, https://github.com/go-gitea/gitea/pull/23901 is the result.

Your fix did not target the right "Remove" button in the `teams/owners` page, but targeted the `org/members` page instead. I put a fix for both, for your case, this was the change that correctly centered it:

https://github.com/go-gitea/gitea/pull/23901/files#diff-1854903b94080fa2568cfa2701a3f1c7a341767e073204c007af034cbcc50402R58

**Pull request closed without merging**